File: arrays_and_hashing/containsDuplicate.java
import java.util.HashSet;

class Solution {
  public boolean containsDuplicate(int[] nums) {
    HashSet<Integer> container = new HashSet<>();
    for(int item : nums) {
      if(container.contains(item))
        return true;
      container.add(item);
    }
    return false;
  }
}

File: arrays_and_hashing/containsDuplicate.cpp
#include <vector>
#include <unordered_set>

class Solution {
public:
  bool containsDuplicate(vector<int>& nums) {
    std::unordered_set<int> container;
    for(size_t i = 0; i < nums.size(); i++) {
      if(container.find(nums[i]) != container.end())
        return true;
      container.insert(nums[i]);
    }
    return false;
  }
};